sdk/eventhub/Azure.Messaging.EventHubs.Processor/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Bugs Fixed
 
+- Changed the approach that the event processor uses to validate permissions on startup to ensure that it uses a non-exclusive consumer as to not interfere with another processor that is already running.  Additionally, an error denying the non-exclusive read is now expected and accepted as proof of a valid connection.
```

Comment:
This seems very implementation specific - do you think we could re-word to focus on the customer observed behavior difference after the fix?

Also, I think the mention of the error could be confusing unless it is observable by developers.

Reply:
That sounds better!

Reply:
Applying manually in a commit for all readmes.